cinema_service/urls.py

```diff
@@ -1,6 +1,9 @@
+from debug_toolbar.toolbar import debug_toolbar_urls
 from django.contrib import admin
 from django.urls import path
+from django.urls.conf import include
 
 urlpatterns = [
     path("admin/", admin.site.urls),
-]
+    path("api/cinema/", include("cinema.urls"), name="cinema"),
+] + debug_toolbar_urls()
```

Comment:
The `debug_toolbar_urls()` should only be included in the URL patterns if the `DEBUG` setting is `True`. Consider wrapping this addition in a conditional check to ensure it is only active during development.